src/Bookings.js

```diff
@@ -1,18 +1,54 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.jsx";
+import NewBookings from "./NewBookings";
+
+// import NewBookings from "./NewBookings.jsx";
 
 const Bookings = () => {
-  const search = searchVal => {
-    console.info("TO DO!", searchVal);
+  const [bookings, setBookings] = useState([]);
+  const [searchData, setSearchData] = useState("");
+  const [loading, setLoading] = useState(true);
+  const [errorOccurred, setErrorOccurred] = useState(false);
+
+  function addBooking(newBooking){
+    const newData = bookings.concat(newBooking)
+    setBookings(newData)
+  }
+
+  const search = (searchVal) => {
+    setSearchData(searchVal.toLowerCase());
   };
+  // `https://cyf-react.glitch.me`;
+  useEffect(() => {
+    fetch(`https://cyf-react.glitch.me/delayed`)
+    .then((res) => {
+      if (res.ok) {
+        return res.json().then((data) => {
+          setBookings(data);
+          setLoading(false);
+        });
+      } else {
+        setErrorOccurred(true);
+      }
+    });
+  }, []);
```

Comment:
There is another way that this fetch function might fail, do you know what it is? 😄